### ISSUE TITLE
WDS trainer presentation (PPT)

### ISSUE THREAD

**Issue by DawnmarieDesJardins** [Microsoft/MCW-Azure-Blockchain#18]
Slides 11 & 22 - images are not flat, no alt-text.
Slide 18 - no alt-text
Slide 12 - alt-text says "Blockchain" not a detailed enough description of the image
Slide 19 - alt-text not detailed enough for image
Accessibility checker comes back with "Missing alt-text, slide 33"  There is no slide 33 visible.  Please check for hidden slides and remove.

**Comment by crpietschmann** [comment]
I updated the alt-text on the slides mentions, and flattened the images noted. I also double checked the deck for hidden slides, and there are not any hidden slides in the deck. There also is no slide 33, so  this error shouldn't be happening in the checker tool.
